calculator-app/src/Components/DisplayResult.js

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import "../Styles/DisplayResultStyle.css"
+ 
+class DisplayResult extends React.Component{
```

Comment:
Because this component doesn't track state, think about updating to be a functional component